reference/features/known_limitations.md

```diff
@@ -1,17 +1,23 @@
-# Known limitations
+# Known Limitations
 
 MeiliSearch has known limitations, some of which may be addressed in future versions.
 
-Some of these limitations have been made by MeiliSearch developers for relevacy and/or performance while others limitations are enforced by LMDB, the key-value store used by MeiliSearch.
+Some of these limitations have been made by MeiliSearch developers for relevancy and/or performance, while others are enforced by [LMDB](/reference/under_the_hood/storage.md), the key-value store used by MeiliSearch.
```

Comment:
Maybe putting `Lightning Memory-Mapped Database Manager` with LMDB in parentheses or the other way around. Or if not here, in the storage under the hood file. Even if we point to the website, it's nice to explain accronym and not expect people to know.

Reply:
I'm not sure that the real name is more understandable than the acronym, I've totally forgotten what `LMDB` means but I still know very well `LMDB`. I will prefer `LMDB` over `Lightning Memory-Mapped Database Manager`. `LMDB` talks to advanced developers but not to everyone, but I think that `Lightning Memory-Mapped Database Manager` is even less known.

Reply:
You both make good points. I'm going to add the full name into /reference/under_the_hood/storage.md, just so that it's written somewhere and will show up if someone searches it for some reason.